documentation/develop/temporary-installation-in-firefox.md

```diff
@@ -115,21 +119,21 @@ Note that in Firefox 48 only, "Reload" does not update the extension's name and
 
 <!-- END: Single Column Body Module -->
 
-<!-- Single Column Body Module -->
+<!-- Single Column Body Module 
```

Comment:
Is this all supposed to be commented out? Looks like updates were made to the copy below that won't be visible.

Reply:
Ooo that should probably be removed, too.